tst/teststandard/grpauto.tst

```diff
@@ -11,6 +11,15 @@
 
 gap> START_TEST("grpauto.tst");
 
```

Comment:
I've run `make teststandard` for this PR. The no-packages version took 1257s and displayed one diff:

```
########> Diff in /home/alexk/GITREPS/gap/tst/teststandard/grpauto.tst:12
# Input is:
START_TEST("grpauto.tst");
# Expected output:

# But found:
########
```

Because it expects the empty line as an output of `START_TEST("grpauto.tst");`. Please delete this empty line,  or even better - leave this empty line intact, but add a comment after it, describing this test case (note that if there would be a comment already, the test would just pass - see my remark below).
___
Remark: It's documented behaviour of `Test` which has to distinguish empty lines as parts of output from empty lines as separators: "To allow for comments in `fname` the following lines are ignored by default: lines at the beginning of `fname` that start with `"#"`, and one empty line together with one or more lines starting with `"#"`. All other lines are considered as GAP output from the preceding GAP input."